test/e2e/primaryoption.spec.ts

```diff
@@ -1,57 +1,124 @@
 /// <reference types="../../cypress/support" />
 
 describe("Primary option page", () => {
-  describe("when user is successfully logged-in", () => {
-    it("should show primary option page once", () => {
-      cy.login();
+  it("should show primary option page once", () => {
+    cy.login();
+    // WHEN I successfully login
+    cy.visit("/anmelden/erfolgreich");
+    // THEN I click continue button
+    cy.contains("a", "Verstanden & weiter").click();
+    // THEN I should see bundesident primary option page
+    cy.contains(
+      "h1",
+      "Möchten Sie sich in wenigen Minuten mit Ihrem Ausweis identifizieren?"
+    );
+    // THEN I click Identifikation mit Ausweis
+    cy.contains("a", "Identifikation mit Ausweis").click();
+    // WHEN I successfully login second time
+    cy.visit("/anmelden/erfolgreich");
```

Comment:
Shouldn't there be a logout in-between?

Reply:
if it's a logout in between, the user will start a fresh session which means they will see the option again. I am testing it under condition that the user is still in the same session

Reply:
Ah ok, the comment tipped me in the other direction. 
```suggestion
    // WHEN I visit the success login page again
    cy.visit("/anmelden/erfolgreich");
```

Reply:
I will replace it in all test suites! thanks for the tip